pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotSchemaRestletResource.java

```diff
@@ -148,9 +150,10 @@ public SuccessResponse updateSchema(
   @ApiResponses(value = {@ApiResponse(code = 200, message = "Successfully updated schema"), @ApiResponse(code = 404, message = "Schema not found"), @ApiResponse(code = 400, message = "Missing or invalid request body"), @ApiResponse(code = 500, message = "Internal error")})
   public SuccessResponse updateSchema(
       @ApiParam(value = "Name of the schema", required = true) @PathParam("schemaName") String schemaName,
+      @ApiParam(value = "Whether to force update the schema if the new schema is not backward compatible") @DefaultValue("false") @QueryParam("force") boolean force,
       @ApiParam(value = "Whether to reload the table if the new schema is backward compatible") @DefaultValue("false") @QueryParam("reload") boolean reload,
       Schema schema) {
-    return updateSchema(schemaName, schema, reload);
+    return updateSchema(schemaName, schema, force, reload);
```

Comment:
Shall we not have `force` parameter?
When do we really expect user to force a backward incompatible change? They might interpret this as "by default backward incompatible changes are not allowed but it can be allowed if this is set to true"